src/test/java/org/opensrp/web/rest/StockResourceTest.java

```diff
@@ -190,11 +214,153 @@ public void testFilter() {
 		assertEquals(expected.size(),actual.size());
 		assertEquals(expected.get(0).getIdentifier(),actual.get(0).getIdentifier());
 	}
+
+	@Test
+	public void testCreateInventory() throws Exception {
+		Authentication authentication = mock(Authentication.class);
+		authentication.setAuthenticated(Boolean.TRUE);
+		SecurityContext securityContext = mock(SecurityContext.class);
+		SecurityContextHolder.setContext(securityContext);
+		when(SecurityContextHolder.getContext().getAuthentication()).thenReturn(getMockedAuthentication());
+		Mockito.doNothing().when(stockService).addInventory(any(Inventory.class), anyString());
+		MvcResult result = mockMvc.perform(post(BASE_URL).contentType(MediaType.APPLICATION_JSON)
+				.content(INVENTORY_PAYLOAD.getBytes()))
+				.andExpect(status().isCreated()).andReturn();
+		assertEquals(result.getResponse().getContentAsString(), "");
+	}
+
+	@Test
+	public void testUpdateInventory() throws Exception {
+		Authentication authentication = mock(Authentication.class);
+		authentication.setAuthenticated(Boolean.TRUE);
+		SecurityContext securityContext = mock(SecurityContext.class);
+		SecurityContextHolder.setContext(securityContext);
+		when(SecurityContextHolder.getContext().getAuthentication()).thenReturn(getMockedAuthentication());
+		Mockito.doNothing().when(stockService).updateInventory(any(Inventory.class), anyString());
+		MvcResult result = mockMvc.perform(put(BASE_URL +"/{id}", "ead1e3ef-b086-4d99-89f5-3dfc5f67709e").contentType(MediaType.APPLICATION_JSON)
+				.content(INVENTORY_PAYLOAD.getBytes()))
+				.andExpect(status().isCreated()).andReturn();
+		assertEquals(result.getResponse().getContentAsString(), "");
```

Comment:
Add an assert that inventory updated is called on service

Reply:
Done